src/router/index.js

```diff
@@ -39,5 +40,10 @@ export default new Router({
       name: 'profile',
       component: Profile,
     },
+    {
+      path: '/dashboard',
+      name: 'dashboard',
+      component: Dashboard,
```

Comment:
Add dashboard to the logged in view of the header.

Reply:
Okay. Will do that once I'm done with all the three components.